frontend/src/metabase/components/Popover/Popover.css

```diff
@@ -44,6 +44,10 @@
   padding: 10px 12px;
 }
 
+.tippy-box[data-theme~="no-padding"] .tippy-content {
```

Comment:
this strikes me as a kind of weird API, maybe providing a comment with a link to the documentation could be helpful down the road?

https://atomiks.github.io/tippyjs/v6/themes/

Reply:
That makes sense, I will add the link in the following PR